frontend/src/views/setting/panel/index.vue

```diff
@@ -413,7 +405,12 @@ const onChangeApiInterfaceStatus = async () => {
         .then(async () => {
             loading.value = true;
             form.apiInterfaceStatus = 'disable';
-            await updateSetting({ key: 'ApiInterfaceStatus', value: 'disable' })
+            let param = {
+                apiKey: form.apiKey,
+                ipWhiteList: form.ipWhiteList,
+                apiInterfaceStatus: form.apiInterfaceStatus,
+            };
+            await updateApiConfig(param)
                 .then(() => {
                     loading.value = false;
                     search();
```

Comment:
There are no significant differences that I can identify between the two versions of the code. However, it's recommended to perform additional checks and analysis based on specific conditions or requirements before moving forward with the release.

If you have further questions or need more detailed feedback specifically about this version, please feel free to ask!